.github/workflows/main.yml

```diff
@@ -49,25 +46,28 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, macos]
-        python: ['3.10', '3.9']
-        mpl-version: ['3.5', 'latest']
+        python: ['3.9', '3.12']
+        mpl-version: ['latest']
+        dist: ['ipympl*.whl']
         include:
-          - python: '3.10'
-            dist: 'ipympl*.tar.gz'
+          # Earliest supported Matplotlib, needs python <= 3.10
           - python: '3.9'
+            mpl-version: '3.5'
             dist: 'ipympl*.whl'
+          # Install from sdist not wheel
+          - python: '3.12'
+            mpl-version: 'latest'
```

Comment:
Are we missing an `os` entry here? I wonder if this is the reason why the workflow does not start

Reply:
Yes, I've corrected this now.

Could you cancel the old CI runs as I don't have permissions to do that?

Reply:
Thanks!